### ISSUE TITLE
[BUG] - Accessibility / Input should have a strong focus indicator when focused with a keyboard

### ISSUE THREAD

**Issue by jguddas** [nextui-org/nextui#918]
### Describe the bug

Input focus is very subtle.

### Steps to Reproduce the Bug or Issue

Focus an input using the keyboard.

### Expected behavior

The standard `cssFocusVisible` indicator could be shown for keyboard focused inputs.

#### Example of a possible solution

https://user-images.githubusercontent.com/25524993/204374701-86afa416-7449-4731-a61f-8a2425945e5e.mp4

### Operating System Version

All

### Browser

Chrome
